docs/source/tutorials_source/platform/tutorial_active_learning.py

```diff
@@ -94,38 +111,29 @@
 # -----------------
 #
 # Import the Python frameworks we need for this tutorial.
+# We need numpy, scikit-learn and lightly as non-standard packages.
+# If you don't have them install them via pip:
+# in terminal: pip install numpy
```

Comment:
code block? :)

Reply:
I put it here, as I assumed that most users will have the frameworks already installed.